udata/frontend/markdown.py

```diff
@@ -1,18 +1,52 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from urlparse import urlparse
+
 import bleach
 import CommonMark
 from html5lib.sanitizer import HTMLSanitizer
 from flask import current_app, Markup
 from werkzeug.local import LocalProxy
 from jinja2.filters import do_truncate, do_striptags
 
+from udata.i18n import _
+
 md = LocalProxy(lambda: current_app.extensions['markdown'])
 
 EXCERPT_TOKEN = '<!--- --- -->'
 
 
+def avoid_mailto_callback(attrs, new=False):
+    """Remove completely the link containing a `mailto`."""
+    if attrs['href'].startswith('mailto:'):
+        return None
```

Comment:
I'm sure to remember it was added because we had cases, it was asked. The use case was to easily allow to put an official contact point in the dataset description. But things evolved since, there are discussions (but discussions are only on the site dataset page which assume the contact point is a registered user).
But let's merge it like that.